lib/class-wp-theme-json-resolver-gutenberg.php

```diff
@@ -323,6 +323,26 @@ public static function get_theme_data( $deprecated = array(), $options = array()
 			if ( current_theme_supports( 'appearance-tools' ) ) {
 				$theme_support_data['settings']['appearanceTools'] = true;
 			}
+
+			// Allow themes to enable link colors via theme_support.
+			if ( current_theme_supports( 'link-color' ) ) {
```

Comment:
I considered "being nice" and include the previous name, experimental-link-color in this condition.
What do you think?